### PR TITLE
MBS-7725 Clarify publishing documentation

### PR DESCRIPTION
**Pull request by eugene-krivobokov**

### CODE REVIEW

**Review by dsvoronin — APPROVED** [review]

**Review by int02h — APPROVED** [review]

File: docs/content/docs/Infrastructure.md [hunk 1]
```diff
@@ -136,41 +136,46 @@ Shared modules between android-test and gradle.
 
 ## Publishing
 
-### Versioning
+### Versioning releases
```
Comment (sergioboi) [inline docs/content/docs/Infrastructure.md]: Release new version

File: docs/content/docs/Infrastructure.md [hunk 2]
```diff
@@ -136,41 +136,46 @@ Shared modules between android-test and gradle.
...
 
 ### CI integration tests against avito
 
-1. Run [Teamcity configuration (internal)](http://links.k.avito.ru/fastCheck) to check pull request builds. 
-1. And/or [This one](http://links.k.avito.ru/fullCheck) to check full set of checks.
-1. You don't need to be bothered about versions here, checks of avito would run against generated version of tools project.
+1 Choose a necessary configuration
+
+- [fast check configuration (internal)](http://links.k.avito.ru/fastCheck) - pull request builds
+- [full check configuration (internal)](http://links.k.avito.ru/fullCheck) - a full set of checks
```
Comment (sergioboi) [inline docs/content/docs/Infrastructure.md]: without release chain

File: docs/content/docs/Infrastructure.md [hunk 3]
```diff
@@ -136,41 +136,46 @@ Shared modules between android-test and gradle.
...
+
+2 Run custom build
 
 {{< hint info>}}
-You can also change build branch if you need to test unmerged code.
-But be careful, Teamcity is tricky about this one:
+If you need to test unmerged code, select a custom build branch.\
```
Comment (sergioboi) [inline docs/content/docs/Infrastructure.md:168]: select where?

File: docs/content/docs/Infrastructure.md (hunk 1 above)
Comment (eugene-krivobokov) [reply docs/content/docs/Infrastructure.md]: Decided to move to gradle.properties

File: docs/content/docs/Infrastructure.md (hunk 3 above)
Comment (eugene-krivobokov) [reply docs/content/docs/Infrastructure.md:168]: Added a screenshot

**Pull request merged**